psalm-baseline.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<files psalm-version="3.9.3@2e4154d76e24d1b4e59e6cc2bebef7790cb9e550">
+  <file src="src/Roave/SecurityAdvisories/Advisory.php">
+    <MixedReturnTypeCoercion occurrences="2">
+      <code>$versionConstraints</code>
+      <code>list&lt;VersionConstraint&gt;</code>
+    </MixedReturnTypeCoercion>
+  </file>
+  <file src="src/Roave/SecurityAdvisories/AdvisorySources/GetAdvisoriesFromFriendsOfPhp.php">
+    <MixedArgument occurrences="1">
+      <code>Yaml::parse(file_get_contents($filePath), Yaml::PARSE_EXCEPTION_ON_INVALID_TYPE)</code>
+    </MixedArgument>
+    <MixedReturnTypeCoercion occurrences="2">
+      <code>SplFileInfo[]</code>
+    </MixedReturnTypeCoercion>
+  </file>
+  <file src="src/Roave/SecurityAdvisories/Component.php">
+    <MixedReturnTypeCoercion occurrences="2">
+      <code>$constraints</code>
+    </MixedReturnTypeCoercion>
+  </file>
+  <file src="src/Roave/SecurityAdvisories/Version.php">
+    <MixedArgumentTypeCoercion occurrences="1">
+      <code>$matches</code>
```

Comment:
The `$matches` resulting from `preg_match()` can probably be typed

Reply:
It can :) Or we could wait for https://github.com/thecodingmachine/safe/issues/175. Do you want me to make the change anyway?

Reply:
We can go ahead as-is here - would suffice to add a `@var` in here

Reply:
fixed